src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -344,27 +344,7 @@ public void InitializeService(ServiceHost serviceHost)
                     prefs.EligibleSkuCategories = GetEligibleSkuCategories("AzureSqlDatabase", parameters.IncludePreviewSkus);
                     resultSet.sqlDbResults = provider.GetSkuRecommendation(prefs, req);
 
-                    if (resultSet.sqlDbResults.Count < parameters.DatabaseAllowList.Count)
```

Comment:
this change makes sense and works well as discussed offline - but do we need to make the same change for MI/VM as well? seems we have the same logic there

Reply:
We need to find the right data to trigger those scenarios, but they will probably fail. We may need to create an item to test and address this.